Source/Core/Core/PowerPC/PowerPC.cpp

```diff
@@ -125,6 +125,9 @@ void DoState(PointerWrap& p)
   p.Do(ppcState.pagetable_base);
   p.Do(ppcState.pagetable_hashmask);
 
+  p.Do(ppcState.reserve);
```

Comment:
The savestate version should be updated.

Reply:
Right, forgot that while rolling back the compatibility code. Just a moment.